compilator/compilator.go

```diff
@@ -682,9 +698,12 @@ func (c *Compilator) BaseImageName() string {
 	return util.SanitizeDockerName(fmt.Sprintf("%s:%s", c.baseCompilationImageRepository(), c.baseCompilationImageTag()))
 }
 
-func (c *Compilator) initPackageMaps(release *model.Release) {
-	for _, pkg := range release.Packages {
-		c.packageDone[pkg.Name] = make(chan struct{})
+func (c *Compilator) initPackageMaps(releases []*model.Release) {
+	for _, release := range releases {
```

Comment:
Here we can range over all packages in all releases and just dump all of them in map, referencing them by their signature.